package.json

```diff
@@ -3,32 +3,49 @@
   "version": "0.0.0",
   "private": true,
   "license": "Apache-2.0",
-  "scripts": {},
+  "scripts": {
+    "build": "yarn ts-node node_modules/.bin/webpack -c ./webpack.config.ts",
+    "dev": "yarn ts-node node_modules/.bin/webpack serve -c ./webpack.config.ts --progress",
+    "ts-node": "ts-node -O '{\"module\":\"commonjs\"}' -I '/node_modules/(?!(@fusion)/)/'"
+  },
+  "workspaces": [
+    "packages/*"
+  ],
   "dependencies": {
     "@openshift-console/dynamic-plugin-sdk": "0.0.15",
     "@openshift-console/dynamic-plugin-sdk-internal": "0.0.7",
+    "@openshift-console/dynamic-plugin-sdk-webpack": "0.0.6",
     "@patternfly/react-core": "4.267.5",
     "react": "^17.0.1",
     "react-dom": "^17.0.1",
     "react-helmet": "^6.1.0",
     "react-i18next": "^11.8.11",
     "react-router": "5.2.0",
     "react-router-dom": "5.2.0",
-    "typescript": "^5.1.3",
-    "ts-node": "5.0.1"
+    "ts-node": "5.0.1",
+    "typescript": "^5.1.3"
   },
   "devDependencies": {
     "@types/node": "^18.0.0",
     "@types/react": "^17.0.37",
     "@types/react-helmet": "^6.1.4",
-    "@types/react-router-dom": "^5.3.2"
+    "@types/react-router-dom": "^5.3.2",
+    "cache-loader": "^4.1.0",
+    "fork-ts-checker-webpack-plugin": "^8.0.0",
+    "thread-loader": "^4.0.2",
+    "ts-loader": "^9.4.3",
+    "webpack": "^5.85.0",
+    "webpack-cli": "^5.1.3",
+    "webpack-dev-server": "^4.15.0"
   },
   "consolePlugin": {
     "name": "fusion-console",
     "version": "0.0.0",
     "displayName": "Fusion Console Plugin",
     "description": "Console Plugin for ODF + Fusion Integrated UI.",
-    "exposedModules": {},
+    "exposedModules": {
+      "features": "@fusion/core/Features"
```

Comment:
file name
```suggestion
      "features": "@fusion/core/features"
```

Reply:
lets decide this for later. I would rather have filenames start with Upper case and folders with lowercase.

Reply:
According to what we discussed, I think this be features